deploy/chart/ingressmonitorcontroller/README.md

```diff
@@ -0,0 +1,94 @@
+|---------------|---------------------------------------------------------------------------|------------------------------------|------------------------------------|
+| useFullName          | Option to use full name containing release name for resources                                                      | `false`                        | `false`                        |
+| matchLabels          | Match labels for deployment selector                                                      | `{}`                        | `{}`
+| deployment.labels          | Labels for deployment                                                      | `{}`                        | `{}`
+| deployment.annotations           | Annotations for deployment                                                      | `{}`                        | `{}`
+| config.labels          | Labels for configmap                                                      | `{}`                        | `{}`
+| config.annotations          | Annotations for configmap                                                      | `fabric8.io/target-platform: kubernetes`                        | `fabric8.io/target-platform: kubernetes`
+| rbac.create          | Option to create rbac resources                                                      | `true`                        | `true`
+| rbac.labels          | Labels for rbac                                                      | `{}`                        | `{}`
+| serviceAccount.create          | Option to create service account                                                      | `true`                        | `true`
+| serviceAccount.labels          | Labels for serviceAccount                                                      | `{}`                        | `{}`
+| serviceAccount.name          | Name of serviceAccount                                                      | `ingressmonitorcontroller`                        | `ingressmonitorcontroller`
+| image.name          | Image of ingressMonitorController                                                      | `stakater/ingressmonitorcontroller`                        | `stakater/ingressmonitorcontroller`
+| image.tag          | Version of ingressMonitorController Image                                                      | `1.0.47`                        | `1.0.47`
+| image.pullPolicy          | Pull policy for image                                                      | `IfNotPresent`                        | `IfNotPresent`
+| providers.name          | Name of the provider                                                      | `UptimeRobot`                        | `UptimeRobot`                        |
+| providers.apiKey        | ApiKey of the provider                                                    | `u956-afus321g565fghr519`            | `your-api-key`                       |
```

Comment:
Seems better to remove the default value for `providers.apiKey` in values.yaml file and set "" here.